docs/process/using-the-staging-environment.md

```diff
@@ -15,25 +15,27 @@ The first thing you'll need to do is create a Firefox account while pointed at t
 ### Desktop
 
 1. First, make sure you are [using a new profile](/ecosystem-platform/docs/process/using-the-staging-environment#create-a-new-firefox-profile).
-2. Now, in `about:config`, set `identity.fxaccounts.autoconfig.uri` to `https://accounts.stage.mozaws.net`.
-3. Restart Firefox. Make sure to use the same profile you setup in step #1.
-4. Now, either login to an existing staging Firefox account, OR create a new Firefox account now that you're pointed to staging.
-5. Once you're logged into your Firefox account, verify that you're actually pointed to staging by [testing a manual Sync](/ecosystem-platform/docs/process/sync-testing#verify-ive-set-things-up-correctly).
+1. Now, in `about:config`, set `identity.fxaccounts.autoconfig.uri` to `https://accounts.stage.mozaws.net`.
```

Comment:
I did not know this cool markdown trick when I first added this file. As long as I'm in here, figured I'd tweak these too.